constants/domain_types.json

```diff
@@ -79,16 +70,27 @@
         "google videos",
         "bing videos",
         "giphy",
-        "getty-images"
+        "getty-images",
+        "and Getty-Images",
+        "Getty-Images",
+        "Commerce",
+        "Earnings Calls"
     ],
-    "Academic and Research": [
+    "Books, Academic & Encyclopedias": [
         "Academic Papers",
         "Biomedical",
         "Education",
         "Exams",
-        "ML Datasets"
+        "ML Datasets",
+        "Books",
+        "Encyclopedias"
+    ],
+    "Government Documents": [
+        "Governments",
+        "Legal",
+        "Parliament"
     ],
-    "Crowdsourced and Human Participants": [
```

Comment:
In speech/video, "Human Participants" often implies N=few, recorded in lab or home contexts, etc. which is quite different from Crowdsourcing in terms of scalability, recruitment of participants, etc. Wondering if "Human Crowdsourced" makes this less clear?